pkg/kritis/cryptolib/verify_detached_test.go

```diff
@@ -0,0 +1,183 @@
+OIssRgtVUtNnngQkgoOwJWxGf0k1R/NrXRsRMReyuW1zT6iftVhClexf5eBLtmqy
+7hXCpy7NNx9486YpF8KJGa1wrV3Ko3R1qHYjGgKd/fjtQ5CcFUZUYgmYQeIkeDO1
+DiQOSYS2xnkUTfV2aHIgFUlrxdU2M50He5Z/jSE49/9IXQac599y2m6irVaVT6VM
+vY6mxSRGgRx3hkoajVrxd3MglIFLO8/FfrOGwZg9bZFIlgHjUiFPPKbn8bSyldMZ
+xNj36t7YC2ZpN6XGzeBnnLfBW1GjGomk3G1eyBUCwRk9nHt9Dsqr/xEauMocHcfA
+EfEm8/fwcp5MB8GOZ3l8p9Rs38yoBFzVbj2SeicFoQTnHe4DrtMvBzGyKsiaKszV
+5UJAPyqm7zPph/Ck++NRQDmJvkhLXcs7QrUG2TBA7veqaLzrU88jY3RDKSIbNtLA
+xBrmQID1ms7TVrCcB6nWrxHJc3FhTP6q/lUL0ziG8fOND9oW6P0YxaXj+1qIxn45
+x2j5oDwsx8ig4/V23Yuofvji2j/VYAY/KAkKJOXqfozbmlMNUCpN6HuQ5yakVFgW
+XanFz99rw4kZ0z70ziCBp68CAwEAAQ==
+-----END PUBLIC KEY-----`
+
+// Signatures created with RSA Keys. Signatures created with rsa.SignPKCS1v15 and rsa.SignPSS methods from crypto/rsa package.
+const rsa2048_256_sig = "DDFzpubahBxVPnKdImjEx8qWssgSezVQm7iQt_VaKhi5SuTpNnryIejmBjOvnDG2u94veHvBANwHPTaH7_m6L07N73yVHaOG86BI6z5cVG-jCriLP5acYKosbsSW9rL7ei8IyVR1JEn0LVaIfNjd4LNMQA5fgTIPQ6zqFfqqbFGAKyTFs6qZoz6JVD253-jpSt_2gBPTbav-wubI2ZxMEtsn6OJZbJf08wGjCwdqs_8qM6-N4DaMlkDooQc3qUyeG3PtL0Oc8wv663YNgB0lMwPfFSTiiHWfqlaA5ipWYxR1zabp3edczblEhAzb7t6Eq3LxdbMJV0GsCiFuqur-PA"
+const rsa2048_256_pss_sig = "fPdgFVY9OZSDTHqt7a74gFdBokpPuiUntWtPk0W2Bnsbbc2LLAG9qvt-z-WrKQ4IKHjq8Grm-asuePTKvIKRrLETgth2tXNT2yRFplmR3F6rwRbIZrcxEtjWLI2d0n0FVNdjdu7ozlJRWSQV666VP3Sq8BnZE6T5KpoSu4FYgNnE_12Y-DBzSmxcMSuEemO7-LBda0aGU8VxLpa5e0dq614BshF56yYLKIgjeNxBZmvTRSxn6sFdNiSdWed2KWsLAiUgwgcfIEQo4wuAXRia3nlW8ti0lsr3nDPURKdc51-jSCF-Ng9iTETlZeoeT5b2RIPRVPvs4mujrkfYJ_Mphw"
+const rsa4096_512_sig = "AAgs8EX5jJ56OcI7V5guArvPbOyLGRyK79_F_1DuubBY-2Q_PdiiTJ2qcfs1TCrVudrJsO-SrUL1sgNpUMl44fcCXHw2SvIecPmdHkkIU6IFtTrBXwie-WUsQqr7kLkhjWejoGdI7bYVzPLT_k_RE8hMDBl1CaQuCMcESVdTqremN2TAax4j4PvbaFYyWDEDM9gGf5ylMzXO6gOw-3HZOrcQ44zNjNhcu6W6xs730TRyGbE9a3q3XyDjNCiikT6U36eds6-m5rDPNMvqin0YJda5ZDOdxge2f9HOu0PHR7WKPtF-65SVHM58c6qzuJHr7jLqG_PkKnBTlwLHZH3G6kZDwl3IepkOkw_esuX-BdzRdS7eFhpr5c20N5VfOYlU_itnEFs850iUkkQ_48vdQC55-hbkYxH9qbYSMbfiVrLxmiI3f5pXmX3MJv1LrHA3XDtBi0dhQpJVYyMfZZmmwZt-oJk-tIzoZb0vBDP5ucCUaiKReiRhCTw07VAH-UY2f5cvRhZ0CW8aZMhkH7JxT1JETov6mRo-kmMsRLKqkTLDjBkrl3YNtRKMQTauCFYCMME4bF4qJxPPBres3LlbAKyNMT0b9V6bpBUrQv247rawl2DEiDUZMDswKN3_kGMfOzW9iCetQYtBRxfkLygocm49UZO3wuNg-deT7Sya7jo"
+const rsa4096_512_pss_sig = "f8gSD9u6XkwsqZYEaQtYdha8MwKNtwhtJRdD4YW-hQkTwPZa3dyj7PRulAI_MQWKCANFAVsqh2sHAP7atBpO79Evccqng_XBTsW6VYxMmHE4R3V7_dBKFHPdt3YE66kufrAy5TzYdAtQTikxT2C8SUb6L_KJg4Byavsx1sPbeRLEQBj8a_glBqfgqmr_4i06hfv9c9a1MGye8SSsaxYfURiGeSRs_mTep5CyE0Xze_ByZeoEEZjYsrpaFVzVsvViNHNcrP1GxsBL8GCz1oPcmAUD1P1nRT_X1kYycuodqpX4Pca3OpBp7kGRdXdBm6UugXLYPoHHZd6ks4NXRrTcY9OieE8s3UmQIwG6cJN4o4j8fmZF68iC50_vlA5rGBtxsdRCcmL9Q9LBZKgKH-FRHagrixAm-T3Ahbf-VpRDodgjz01Lwz8cVMfcZJ3SeHPoAvgOO4fGRRTQMpzKmDWNBlx2Nj3WXyIOeJEkg_2kJ7GEt5FspiXPGZsmU1XIJxblCZ_h1dpbQf0ow4l_fmQyAv6q2Gxw0J2I3TmaISsQfwHCpD07FBcp4hLrVrVdRIeRLvhYPcWUaWYn09whhwwpa0s2AJeKEKplbJuJRX322kyfzFsiAAcECWQEc0fz9gNw9aEaWhuUKevS_NpKx7rLZjkFzAYLQKQDme2mbm7OkC8"
```

Comment:
Should there also be signatures `rsa4096_256_sig` and `rsa4096_256_pss_sig` to cover SignatureAlgorithms `RsaSignPkcs14096Sha256` and `RsaPss4096Sha256`, respectively?

Reply:
done.